spinn_utilities/config_holder.py

```diff
@@ -173,10 +180,21 @@ def get_config_bool(section, option):
 def set_config(section, option, value):
     """ Sets the value of a config option.
 
+    This method should only be called by the simulator or by unittests
+
     :param str section: What section to set the option in.
     :param str option: What option to set.
     :param object value: Value to set option to
     """
+    if __config is None:
+        if __unittest_mode:
+            load_config()
+        else:
+            # The actual error is that load_config should be called before
+            # set_config but this discourages the use outside of unittests
+            raise Exception(
+                "set_config should only be called by unittests "
+                "which should have called reset_configs")
```

Comment:
... which should have called unittest_setup I think

Reply:
good spot. fixed